### ISSUE TITLE
Usersnap Feedback - I would be a little bit more explicit in explaining how your ability to measure

### ISSUE THREAD

**Issue by jonlong** [remedyplan/remedyplan.github.io#23]
**Sender**: iglomski@yahoo.com
[Open #4 in Usersnap Dashboard](https://usersnap.com/a/#/company/p/remedyplan-com-f-d5f596d2/4)
**Notes**:
 - (1) I would be a little bit more explicit in explaining how your ability to measure cancer's ability to spread allows you to develop drugs. I assume adding compounds to your spreading assay and seeing if the compounds reduces that ability. This connection may not be obvious to everyone.

![Usersnap Feedback - I would be a little bit more explicit in explaining how your ability to measure](https://usersnappublic.s3.amazonaws.com/2015-06-11/17-39/2870b76b-6ca2-4dea-bb34-34e9fe4132d5.png)

**Browser**: Chrome 43 (Windows 7)
**Referer**: [https://remedyplan.com/#faq](https://remedyplan.com/#faq)
**IP**: 184.0.211.197
**Screen size**: 1680 x 1050 **Browser size**: 1356 x 834

Powered by [usersnap.com](https://usersnap.com/?utm_source=github_entry&utm_medium=web&utm_campaign=product)

**Comment by gregcrimmins** [comment]
Yes, what Ian said is what we're doing. Should we go into more detail, or are most people not going to want to hear the details?
